internal/net/grpc/logger/logger_test.go

```diff
@@ -101,37 +109,15 @@ func Test_logger_Info(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           args: nil,
-		       },
-		       fields: fields {
-		           v: 0,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           args: nil,
-		           },
-		           fields: fields {
-		           v: 0,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "info log the message",
```

Comment:
please fix according to the guideline :)

Reply:
```suggestion
			name: "info success to log the message",
```